tendrl/monitoring_integration/grafana/utils.py

```diff
@@ -1,56 +1,42 @@
+import __builtin__
 import os
 import socket
-import sys
 
 
 from ruamel import yaml
 
 
 from tendrl.monitoring_integration.grafana import exceptions
 from tendrl.monitoring_integration.grafana import config
+from tendrl.commons.utils import log_utils as logger
 
 
-def get_conf(file_name):
+def get_conf():
 
     try:
-        yaml_config = _load_config(file_name)
 
-        conf = config.Config()
         # Graphite and Grafana will be running on localhost
-        conf.grafana_host = yaml_config.get('grafana_host', "localhost")
-        conf.grafana_port = yaml_config.get('grafana_port', 3000)
-        conf.datasource_host = yaml_config.get('datasource_host', "localhost")
-        conf.datasource_port = yaml_config.get('datasource_port', 10080)
-
-        # Datasource name in Grafana
-        conf.datasource_name = yaml_config.get('datasource_name', "Graphite")
+        NS.config.data["grafana_host"] = "localhost"
```

Comment:
@r0h4n This function (get_conf) sets the information required for uploading the dashboard but are not to be requested by the user and hence are not present in /etc/tendrl/monitoring-integration/monitoring-integration.conf.yaml
This function will also be used in future to insert values that are not present in monitoring-integration.conf.yaml
I have already removed the data that is present in monitoring-integration.conf.yaml and is added to NS.conf by commons lib.
i will remove the redundant functions like _load_config